internal/batches/docker/info.go

```diff
@@ -10,6 +10,31 @@ import (
 	"github.com/sourcegraph/src-cli/internal/exec"
 )
 
+// CurrentContext returns the name of the current Docker context (not to be
+// confused with a Go context).
+func CurrentContext(ctx context.Context) (string, error) {
+	dctx, cancel, err := withFastCommandContext(ctx)
+	if err != nil {
+		return "", err
+	}
+	defer cancel()
+
+	args := []string{"context", "inspect", "--format", "{{ .Name }}"}
```

Comment:
Wow, it makes me happy that this is so straightforward to get from Docker! 😄

Reply:
I suspected the hardest part of this ticket was going to be detecting this case, so this was a very pleasant surprise.